.github/workflows/publish.yaml

```diff
@@ -0,0 +1,30 @@
+name: 'Publish to Maven Central'
+
+on: workflow_dispatch
+
+jobs:
+  publish:
+    name: 'Publish'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    timeout-minutes: 120
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 11
```

Comment:
What does it mean to specify JDK version 11 here?

Reply:
It will install jdk11 for the build image.
I think I should change it to 8 | 11 | 17 like here:

https://github.com/actions/setup-java/releases/tag/v3.5.0

I think we already have multi-jdk release build and it should work.